src/services/refactors/convertImport.ts

```diff
@@ -153,10 +176,8 @@ namespace ts.refactor {
                 if (isShorthandPropertyAssignment(id.parent)) {
                     changes.replaceNode(sourceFile, id.parent, factory.createPropertyAssignment(id.text, access));
                 }
-                else if (isExportSpecifier(id.parent) && !id.parent.propertyName) {
-                    if (!neededNamedImports.some(n => n.name === element.name)) {
-                        neededNamedImports.push(factory.createImportSpecifier(element.propertyName && factory.createIdentifier(element.propertyName.text), factory.createIdentifier(element.name.text)));
-                    }
+                else if (isExportSpecifier(id.parent)) {

```

Comment:
Old code disregarded named export clauses that aliased the reference to the named import, so code such as 
```ts
import { a } from "m";
export { a as b };
```
was in fact incorrectly refactored into
```ts
import { a } from "m";
export { m.a as b };
```